index.html

```diff
@@ -14,6 +14,66 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <header>  
+    <h1>Natalie Peterson</h1>
+    <h2>child & friend</h2>
+  <nav>
+    <a href="#">About</a>
+    <a href="https://objective-bose-29c02b.netlify.app/">Portfolio</a>
+    <a href="#">Contact</a>
+  </nav>
+</header>
+<section>
+  <div>
+  <h3>Experience</h3>
+  <h5>Cookie Connoisseur</h5>
+  <h6>feb 1996 - current</h6>
+  <p>travel to all places to get the best cookies.</p>
+  <p>knows when a cookie business will be successful</p> 
+</div>
+<div>
+  <h5>Sparkling Water Expert</h5>
+  <h6>oct 2010 - current</h6>
+  <p>tasting pro with knowledge from la croix to mountain valley. </p>
+  <p>help people find the sparkling water that best meets thier taste buds needs</p>
+</div>
+<div>
+  <h5>Veggie Roaster</h5>
+  <h6>aug 2006 - current</h6>
+  <p>Professional veggie roaster specializing in brocoli, burssel sprouts, and onions</p>
+  <p>Extensive experience with broil mode, foil, and olive oil bases</p>
+</div>
+</section>
+
+<section>
+    <h3>Education</h3>
+  <div>
+    <h5>House of Crumbs</h5>
+    <h6>feb 1996 - dec 2009</h6>
+    <p>graudated magna cookie</p>
+  </div>
+  <div>
+    <h5>Bubbly Static Water</h5>
+    <h6>feb 1996 - oct 2019</h6>
+    <p>blackbelt</p>
+  </div>
+  <div>
+  <h5>Brocolli Parade</h5>
+  <h6>feb 1996 - nov 2014 </h6>
+  <p>top veggie chef 10 years in a row</p>
+</div>
+
+</section>
+<section>
+  <h3>Skills</h3>
+  <h5>Eating Cookies</h5>
```

Comment:
Since these are list items under a skills list, I'd suggest using `<ul>` and `<li>` tags for accessibility and semantic HTML.